ports/raspberrypi/boards/pimoroni_plasma2350/pins.c

```diff
@@ -0,0 +1,52 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
+
+    { MP_ROM_QSTR(MP_QSTR_SW_A), MP_ROM_PTR(&pin_GPIO12) },
+    { MP_ROM_QSTR(MP_QSTR_SW_BOOT), MP_ROM_PTR(&pin_GPIO22) },
```

Comment:
In our schematic we call GPIO22 `USER_SW`. I also see in the other board defs that you add a `BUTTON` def, though those other boards only have the one button.

```python
    { MP_ROM_QSTR(MP_QSTR_USER_SW), MP_ROM_PTR(&pin_GPIO22) },
    { MP_ROM_QSTR(MP_QSTR_BUTTON), MP_ROM_PTR(&pin_GPIO22) },
```

Reply:
Made 12 `BUTTON` and added `BOOT` and `USER_SW` to 22.